scripts/glusterfs_pool.sh

```diff
@@ -0,0 +1,13 @@
+#/bin/bash
+
+HOSTLIST=$1
+
+systemctl start glusterd
+systemctl status glusterd
+
+if [ "$PSSH_NODENUM" = "0" ]; then
+   for host in `cat ~hpcadmin/azhpc_install_config/hostlists/$HOSTLIST`
```

Comment:
username and config file name hardcoded, maybe the hostlist full path should be provided instead ? @edwardsp what do you think ?

Reply:
the scripts are run from inside the temp directory - you could just use "cat hostlists/$HOSTLIST".  But, better still, you could just set the arg in the config as "$(<hostlists/glusterfs)" and then you would get it as the first arg.  You would only then need to iterate as follows:
for host in $HOSTLIST; do ...

Reply:
Thanks Paul and Xavier, I made the changes you suggested, looks cleaner now.